src/objects.rs

```diff
@@ -1578,7 +1578,7 @@ pub struct UsersShared {
 pub struct ChatShared {
     pub request_id: i32,
 
-    pub chat_id: u64,
+    pub chat_id: i64,
```

Comment:
https://github.com/ayrat555/frankenstein/blob/cc469d643c218ab9d40603ed5de31196a934d4cf/src/api_params.rs#L136
```suggestion
    pub chat_id: ChatId,
```

Reply:
Good point. Then I guess i64 is the way to go.